execution_workflows/APA-Scan/README.md

```diff
@@ -0,0 +1,24 @@
+# APA-Scan
```

Comment:
I think this should focus on how to run the execution workflow rather than the tool. In particular, it should include a description of the various config files, sample sheets etc